src/implementations/react/src/playground/sections/DropdownSection.js

```diff
@@ -2,82 +2,47 @@ import React, { Component } from 'react';
 import PlaygroundSection from '../PlaygroundSection';
 import { Dropdown } from '../../hig-react';
 
-const Option = Dropdown.Option;
-
-const optionData = [
-  { label: 'text option label', value: 'test option value', selected: true },
-  {
-    label: 'text option label 1',
-    value: 'test option value 1',
-    selected: false
-  }
-];
-
 class DropdownSection extends Component {
-  constructor() {
-    super();
-    this.state = {
-      open: false
-    };
-  }
-
-  setSelectedValue = (selectedOption) => {
-    this.setState({
-      selectedOptionLabel: selectedOption.label,
-      open: false
-    });
-  };
-
-  logEvent(event, higElement) {
-    let messageParts = [
-      `${higElement.constructor.name} triggered an ${event.type} event`
-    ];
-    if (event.target.value !== undefined) {
-      messageParts = messageParts.concat(`: ${event.target.value}`);
-    }
-    console.log(messageParts.join(''));
-  }
-
-
-  openDropdown = () => {
-    this.setState({ open: true });
-  };
-
-  closeDropdown = () => {
-    this.setState({ open: false });
-  };
-
   render() {
     return (
       <PlaygroundSection title="Dropdown">
         <Dropdown
-          instructions="test instructions"
-          label="Dropdown label"
-          selectedOptionLabel={this.state.selectedOptionLabel}
-          placeholder="Select an option"
-          open={this.state.open}
-          onTargetClick={this.openDropdown}
-          onClickOutside={this.closeDropdown}
-          onBlur={this.logEvent}
-          onFocus={this.logEvent}
-          onKeypress={this.logEvent}
-          required={'This field is required'}
-        >
-          {optionData.map(option => (
-            <Option
-              label={option.label}
-              value={option.value}
-              selected={
-                option.label === this.state.selectedOptionLabel
+          label="Controlled Dropdown"
+          instructions="instructions for regular Controlled dropdown"
+          placeholder="placeholder for regular Controlled dropdown"
+          options={
+            [ 
+              { 
+                label: "foo", 
+                value: "foo value" 
+              }, { 
+                label: "bar", 
+                value: "bar value" 
+              }
+            ]
+          }
+          value="bar value"
```

Comment:
A controlled field with a fixed value like this should not change when the user selects something. It should be stuck on `bar` no matter what. 

I'll submit another PR showing what I mean.